feconf.py

```diff
@@ -213,7 +213,7 @@
 # incompatible changes are made to the states blob schema in the data store,
 # this version number must be changed and the exploration migration job
 # executed.
-CURRENT_STATE_SCHEMA_VERSION = 39
+CURRENT_STATE_SCHEMA_VERSION = 38
```

Comment:
Just to confirm, the schema upgrade wasn't part of the previous release right? The data in prod wouldn't be affected by this right?

Reply:
Yup that's right!